package.json

```diff
@@ -229,8 +165,8 @@
     "babel-core": "^6.24.1",
     "babel-eslint": "^7.2.3",
     "babel-gettext-extractor": "git+https://github.com/muffinresearch/babel-gettext-extractor.git#0d39d3882bc846e7dcb6c9ff6463896c96920ce6",
+    "babel-jest": "20.0.3",
```

Comment:
I did look for a config setting but there doesn't seem to be one. There's only one setting to cover both prod and dev prefs.